Source/Core/DolphinQt/DolphinEmu.entitlements

```diff
@@ -13,5 +13,7 @@
 	<!-- This is needed to use adhoc signed linked libraries -->
  	<key>com.apple.security.cs.disable-library-validation</key> 
   	<true/>
+    <key>com.apple.security.get-task-allow</key>
```

Comment:
Is there a reason why this is here? We should be avoiding this entitlement if possible. (Plus, the notary service may reject the app if this is present.)

Reply:
@OatmealDome should we drop this change when merging Qt6?
i'm not familiar with xcode and such, but from googling it looks like it was probably just added by mistake?

Reply:
@shuffle2 It should be dropped, yeah. If we need it, something is probably very wrong. (macOS requires processes to have this entitlement to allow a debugger to be attached to the process, so I guess this was added for testing.)